auto-configurations/mcp/spring-ai-autoconfigure-mcp-server/src/test/java/org/springframework/ai/mcp/server/autoconfigure/McpServerAutoConfigurationIT.java

```diff
@@ -95,6 +100,7 @@ void asyncConfiguration() {
 				assertThat(properties.getVersion()).isEqualTo("2.0.0");
 				assertThat(properties.getInstructions()).isEqualTo("My MCP Server");
 				assertThat(properties.getType()).isEqualTo(McpServerProperties.ServerType.ASYNC);
+				assertThat(properties.getRequestTimeout().getSeconds()).isEqualTo(30);
```

Comment:
I wonder how this can work without having the AsyncSpecification not set with the request timeout at the autoconfiguration level.

Reply:
Got it, it just checks the server properties not the async configuration itself.